test/extract_whiteboard_test.js

```diff
@@ -0,0 +1,24 @@
+import extractWhiteboardTags from '../src/utils/extractWhiteboard';
+
+describe('extract whiteboard tags', () => {
+  it('should get the language', () => {
+    const whiteboard = '[lang=js][lang=xul]';
+    const result = extractWhiteboardTags(whiteboard);
+    expect(result).toEqual(['lang=js', 'lang=xul']);
+  });
+  it('should ignore other whiteboard tag besides language', () => {
+    const whiteboard = '[good-first-bug]';
+    const result = extractWhiteboardTags(whiteboard);
+    expect(result).toEqual([]);
+  });
+  it('should handle comma tag', () => {
+    const whiteboard = '[lang=js,xul]';
+    const result = extractWhiteboardTags(whiteboard);
```

Comment:
`whiteboard` is not used multiple times, so you can just inline it.

```js
const result = extractWhiteboardTags('[lang=js,xul]');
```

Perform this change for the other test cases as well.